Seungwoon/bookStudy/29_toothbrush.py

```diff
@@ -0,0 +1,19 @@
+# enroll = 각 판매원 이름 // referral = 각 판매원의 추천인 // seller = 판매자 // amount = 판매량
+# 칫솔 1개당 100원 -> amount * 100 = 얻은 수익
+# 추천인 없으면 '-'
+# 목적: 각 판매원이 받은 이득 리스트 출력
+# dt1 = {회원 : 추천인} // dt2 = {회원 : 금액}
+# dt2가 정답틀
+def solution(enroll, referral, seller, amount):
+    dt1 = dict(zip(enroll, referral)) # {회원 : 추천인}
+    dt2 = dict(zip(enroll, [0]*len(enroll))) # {회원 : 초기값 0원}
+
+    for i in range(len(seller)):
```

Comment:
```suggestion
    for i, current_name in enumerate(seller):
```
enumerate를 활용하는 것이 좀 더 Pythonic 할 것 같네요

Reply:
enumerate를 사용하면 value값까지 불러오는 불필요한 연산이되지 않을까요?

Reply:
어차피 Line 13에서 seller[i]를 호출하고 있기 때문에 불필요한 연산은 아닌 것 같아요